lib/fp.js

```diff
@@ -2,32 +2,29 @@
 
 const { safeCallback } = require('./callbacks');
 
-const partial = (
-  // Partially apply arguments to function
-  fn, // function
-  ...args // array, argumants
-  // Returns: function
-) => (...rest) => fn(...args.concat(rest));
-
-const omap = (
-  // Map object fields with provided function
-  mapFn, // funtion, to appy to every field value
-  obj // object, which fields used for mapping
-  // Returns: object, with same reference but with transformed fields
-) => {
+// Partially apply arguments to function
+//   fn - <function>
+//   args - <array>, arguments to be applied
+// Returns: function(...rest)
+//   rest - <array>, arguments
```

Comment:
I still don't think `<array>` conveys the meaning of `rest` here.
What if we have special case like we have for an array (`<string[]>`) but instead it will be `<string...>`, wdyt?

Reply:
But what about something like this: https://github.com/metarhia/common/blob/comments-refactor/lib/data.js#L180? Will we have `<Array[]>`, `<Array...>` or `<Array[]...>`?